packages/db/src/translate/executions.ts

```diff
@@ -0,0 +1,71 @@
+import Knex from 'knex';
+
+import { ExecutionRecord } from '@cumulus/types/api/executions';
+import { PostgresExecution } from '../types';
+// TODO move to common location
+const {
+  getParentExecutionCumulusId,
+  getAsyncOperationCumulusId,
+  getCollectionCumulusId,
+} = require('../../api/lambdas/sf-event-sqs-to-db-records/utils');
```

Comment:
My hope is that these can come from the new PG models for Executions, Async Operations, and Collections. Looking at that now as part of merging those changes.